src/components/pages/Books.js

```diff
@@ -0,0 +1,11 @@
+import Navbar from '../Navbar';
+
+const Books = () => (
+  <>
+    <Navbar />
+    <div>This is Books page</div>
+  </>
+
+);
+
+export default Books;
```

Comment:
- [ ] While you create some components for the Books page :+1:, as a project requirement, the Books page should render a list of books using the `<Book>` component. This component should take the book data as a prop and: 
- Renders the title and the author
- Has a button for removing books (the button wouldn't remove the books *yet*, this is just a placeholder for now)

Also, there should be another component that contains a form to create new books with inputs for the title and the author of the book (the button wouldn't add the book *yet*, this is just a placeholder for now).

**Your Books page**

![Screenshot from 2023-05-05 03-11-35](https://user-images.githubusercontent.com/105117832/236409548-41998184-198a-47b4-8a56-407ffd342be5.png)